tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py

```diff
@@ -15,8 +19,8 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('config', '0025_update_device_key'),
-        ('openwisp_users', '0007_unique_email'),
+        migrations.swappable_dependency(settings.CONFIG_DEVICE_MODEL),
+        ('openwisp_users', '0011_user_first_name_150_max_length'),
```

Comment:
@codesankalp can we change this dependency with AUTH_USER_MODEL?
```suggestion
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```